config/sync/user.role.site_editor.yml

```diff
@@ -0,0 +1,53 @@
+uuid: b35dd6b6-930d-4b4c-aeb2-443a0ba6fb59
+langcode: en
+status: true
+dependencies: {  }
+id: site_editor
+label: 'Site Editor'
+weight: -2
+is_admin: null
+permissions:
+  - 'access administration pages'
+  - 'access content overview'
+  - 'access contextual links'
+  - 'access environment indicator'
+  - 'access file_browser entity browser pages'
+  - 'access files overview'
+  - 'access image_browser entity browser pages'
+  - 'access media overview'
+  - 'access media_browser entity browser pages'
+  - 'access patterns page'
+  - 'access video_browser entity browser pages'
+  - 'administer main menu items'
+  - 'administer menu'
+  - 'cancel account'
+  - 'change own username'
+  - 'create file media'
+  - 'create image media'
+  - 'create media'
+  - 'create stanford_page content'
+  - 'create url aliases'
+  - 'create video media'
+  - 'delete any stanford_page content'
+  - 'delete media'
+  - 'delete own file media'
+  - 'delete own image media'
+  - 'delete own stanford_page content'
+  - 'delete own video media'
+  - 'delete stanford_page revisions'
+  - 'dropzone upload files'
+  - 'edit any stanford_page content'
+  - 'edit own file media'
+  - 'edit own image media'
+  - 'edit own stanford_page content'
+  - 'edit own video media'
+  - 'perform batch updates entity usage'
```

Comment:
all roles dont really need this. It just bulk updates usages data.

Reply:
GTK, TY.